Models/ErrorViewModel.cs

```diff
@@ -1,4 +1,4 @@
-namespace StoreManagement.Models
+namespace StoresManagement.Models
 {
     public class ErrorViewModel
     {
```

Comment:
if not required do not put logic into data transport models.
I reffer to `public bool ShowRequestId => !string.IsNullOrEmpty(RequestId);`

Reply:
to be cleared.